content/community/presentations/_index.en.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Presentations from Altinn Coffee
-linktitle: Altinn Coffee
+title: Presentations and Altinn Coffee
+linktitle: Presentations
 description: The Altinn-kaffe meeting series is a low threshold meeting series, where app owners (and potential app owners) can learn more about what we're doing in Altinn 3.
```

Comment:
Should we update the description now that the title changes from Altinn Coffee to Presentations?